src/components/footer/footer.js

```diff
@@ -22,35 +23,34 @@ const socialLinks = [
 
 const Footer = () => {
   return (
-    <footer className="custom-footer">
-      <div className="custom-footer__logo">
-        <Logo />
-      </div>
-      <div className="links-to-pages">
-        {links.map(link => (
-          <Link key={link.id} to={link.path} className="nav-link link">
-            <i className="link-circle fas fa-circle link-text" />
-            {link.name}
-          </Link>
-        ))}
+    <footer className="footer__container">
+      <div className="footer__links_and_logo_container">
```

Comment:
Please don't use prepositions in classnames. In this case there is no problem to create wrapper div only for Link and specify simplified name

Reply:
Done